src/bibliography/infrastructure/BibliographyRepository.test.ts

```diff
@@ -28,13 +28,13 @@ const testData: TestData[] = [
   ],
   [
     'search',
-    [author, year, title],
+    [[author, year, title]],
     apiClient.fetchJson,
     [entry],
     [
-      `/bibliography?author=${encodeURIComponent(
-        author
-      )}&title=${encodeURIComponent(title)}&year=${encodeURIComponent(year)}`,
+      `/bibliography?0=${encodeURIComponent(author)}&1=${encodeURIComponent(
+        year
+      )}&2=${encodeURIComponent(title)}`,
```

Comment:
Weren't you going to use an "array" parameter? This is same as "first", "second", third" but with different names.

Reply:
But this is how an array gets encoded automatically as url

Reply:
Have you meant passig the array like this (as an extra parameter)
```
...
[
    `/bibliography`, [author, year, title], true,
]
...

Reply:
Like described here: https://falcon.readthedocs.io/en/stable/api/request_and_response.html#falcon.Request.params

, but if the parsing is done in the backend, it does not matter.